src/models/Illness.ts

```diff
@@ -0,0 +1,14 @@
+
+import { IllPerson } from './fields/IllPerson';
+
+import { Attachment } from 'app/models/Attachment';
+
+export interface Illness {
```

Comment:
These data models are not part of the ticket, remove for better traceability.

Reply:
Sure.